install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -58,6 +62,8 @@ spec:
     listKind: ServiceEntryList
     plural: serviceentries
     singular: serviceentry
+    shortNames:
+    - se
```

Comment:
this is common and would be used.

Reply:
I have checked that these short names are not used by Kubernetes. If they are used by other tools, I think users should clearly specify the groups.